backend/migrations/versions/799a2343955e_fix_moduletype_enum.py

```
@@ -0,0 +1,118 @@
+    "CHALLENGE_SOLUTION",
+    "BONUS_CHALLENGE",
+    "BONUS_SOLUTION",
+    name="moduletype",
+)
+
+
+def upgrade():
+    # Create a temporary column to hold the new enum values
+    with op.batch_alter_table("modules", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column("module_type_temp", new_module_type, nullable=True)
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)